### ISSUE TITLE
Ep7 Tracking a species: exercise too difficult?

### ISSUE THREAD

**Issue by gcapes** [swcarpentry/shell-novice#982]
http://swcarpentry.github.io/shell-novice/07-find/index.html#tracking-a-species

This exercise doesn't really target the learning objectives of the episode. The main concepts are pipes and filters, and interpreting the man pages to learn about options. Also there is the requirement to implement script arguments from the previous episode.

Given the man pages are required for both `grep` and `cut`, this exercise should have much more scaffolding. Perhaps splitting into steps would help?

**Comment by amanda-tan** [comment]
The learning objectives for Episode 7 are grep and find. Could you be referring to Episode 4 which is Pipe and Filters? 

I think referring to a previous episode can help refresh memory.

**Comment by gcapes** [comment]
I have now edited my comment to improve clarity -- hope that helps.

**Comment by shervinsahba** [comment]
Agreed that the goal isn't clear. The recursive grep makes this tricky. It's not obvious that we are searching the entire tree at first, and a potential hang up would be with `cut -d : -f 2`. The colon delimiter is difficult to interpret if you've yet to try a recursive grep.

Maybe clarify

> She wants to write a shell script that takes a species as the first command-line argument and a directory as the second argument. The script should return one file called species.txt containing a list of dates and the number of that species seen on each date. For example using the data shown above, rabbit.txt would contain:

with instructions

> She wants to write a shell script that takes a _species_ as the first command-line argument and a _directory_ as the second argument. The script should return one file called species.txt containing a list of dates and the number of that species seen on each date. **That is, the script needs to recursively search through the directory for the given species, filter which files contain the species, and then filter the desired columns from the datafile, redirecting the output to species.txt.** For example using the data shown above, rabbit.txt would contain:
